alert-common/src/main/java/com/synopsys/integration/alert/common/channel/DistributionChannel.java

```diff
@@ -64,7 +64,7 @@ public void sendAuditedMessage(DistributionEvent event) throws IntegrationExcept
             auditUtility.setAuditEntrySuccess(event.getAuditIds());
         } catch (IntegrationRestException irex) {
             auditUtility.setAuditEntryFailure(event.getAuditIds(), irex.getMessage(), irex);
-            logger.error("{} : {}", irex.getHttpStatusCode(), irex.getHttpStatusMessage());
```

Comment:
What is the specific error SonarCloud is complaining about here? This may be a suggestion we want to ignore.

Reply:
I undid this particular change, but the issue is SonarCloud wants you to either log and handle the exception OR re-throw the exception but it does not want you to do both. In this case we want to log the extra fields from the IntegrationRestException so we will ignore this smelly smell.